### PR TITLE
BACKLOG-16374: Use CodeSignTool for code signing

### PR DESCRIPTION
**Pull request by gflores-jahia**
## JIRA

https://jira.jahia.org/browse/BACKLOG-16374

## Description

Switch to using CodeSignTool for batch signing installers https://www.ssl.com/guide/esigner-codesigntool-command-guide/#ftoc-heading-7

### CODE REVIEW

File: src/commands/code-signer.yml [hunk 1]
```diff
@@ -47,42 +38,25 @@ parameters:
...
-          mv $INSTALLER_FILE ${INSTALLER_FILE}.unsigned
-
-          echo "Downloading signed installer file $INSTALLER_FILE from ${<< parameters.download_signer_url >>}..."
-          TOTP_TOKEN=`oathtool --totp -b -d 6 ${<< parameters.totp_secret >>}`
-          curl -s --location --request POST "${<< parameters.download_signer_url >>}" --output "${INSTALLER_FILE}" --header "$auth_header" --header "Content-Type: application/json" --header "Content-Transfer-Encoding: application/json" --data-raw "{\"id\": \"$SIGNED_FILE_ID\", \"otp\":\"$TOTP_TOKEN\"}"
+          cp $INSTALLER_FILE /tmp/unsigned-artifacts
         done
+        cd CodeSignTool-v1.2.0
+        printf `oathtool --totp -b -d 6 $TOTP_SECRET` | ./CodeSignTool.sh batch_sign -input_dir_path=/tmp/unsigned-artifacts -password=$SIGNER_PASSWORD -output_dir_path=/tmp/artifacts -username=$SIGNER_USERNAME -credential_id=$CREDENTIAL_ID
+        rm -rf /tmp/unsigned-artifacts
```
Comment (rknj) [inline src/commands/code-signer.yml:62]: we do not keep them?

Comment (cedmail) [reply src/commands/code-signer.yml:62]: What will be the point of keeping them?

Comment (rknj) [reply src/commands/code-signer.yml:62]: in case the job fails, we can take them and sign them manually

Comment (gflores-jahia) [reply src/commands/code-signer.yml:62]: The previous jobs should have unsigned artifacts for download

**Review by rknj — APPROVED** [review]

**Pull request merged**